pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -193,6 +253,10 @@ func (alloc *allocateAction) Execute(ssn *framework.Session) {
 		} else {
 			stmt.Discard()
 		}
+
+		if namespaceOrderEnabled {
```

Comment:
We would enter the place that we have a huge **Execute** method that is much too long 😄

Reply:
refactor action Allocate, remove all `namespaceOrderEnabled` in it.